src/org/freenetproject/freemail/wot/WoTConnectionImpl.java

```diff
@@ -41,14 +41,16 @@
 class WoTConnectionImpl implements WoTConnection {
 	private static final String WOT_PLUGIN_NAME = "plugins.WebOfTrust.WebOfTrust";
 	private static final String CONNECTION_IDENTIFIER = "Freemail";
+	private static int connectionCounter;
 
 	private final PluginTalker pluginTalker;
 
 	private Message reply = null;
 	private final Object replyLock = new Object();
 
 	WoTConnectionImpl(PluginRespirator pr) throws PluginNotFoundException {
-		pluginTalker = pr.getPluginTalker(new WoTConnectionTalker(), WOT_PLUGIN_NAME, CONNECTION_IDENTIFIER);
+		pluginTalker = pr.getPluginTalker(new WoTConnectionTalker(), WOT_PLUGIN_NAME,
```

Comment:
I really hate how we fucked up FCP so hard that every client has to circumvent things it was meant to prevent. :(